cmd/cmd.go

```diff
@@ -324,6 +325,18 @@ func RunHandler(cmd *cobra.Command, args []string) error {
 	}
 	opts.Format = format
 
+	keepAlive, err := cmd.Flags().GetString("keepalive")
```

Comment:
```suggestion
	keepAlive, err := cmd.Flags().GetString("keep-alive")
```

I think this is ok as is, but would suggest `keep-alive` to be consistent with the api `keep_alive` – although flag styling is matter of taste over consistency

Reply:
I thought about this, but we have `--nowordwrap` as one of the other options for `run`, so it felt weird going to kebab case here.